### PR TITLE
chore: Log type of event, which could not be decrypted

### PR DESCRIPTION
**Pull request by bennyn**
Added `eventType` to decryption error logs.

### CODE REVIEW

**Comment by bennycode** [comment]
I need to identify the event which could not be decrypted. 

Given the fact that they are all of type `'conversation.otr-message-add'`, which information would you add to distinguish between multiple OTR messages?

The common envelope is:

> [ "type"         .= <event type>                          -- event type
, "conversation" .= <uuid>                                -- conv id
, "from"         .= <uuid>                                -- user id
, "time"         .= <time utc>                            -- format: "2018-02-21T22:47:18.773Z"
, "data"         .= <data dependent on the event type>
]

**Comment by gregor** [comment]
Timestamp is the only unique information you will find

File: app/script/cryptography/CryptographyRepository.js [hunk 1]
```diff
@@ -470,7 +470,7 @@ z.cryptography.CryptographyRepository = class CryptographyRepository {
     }
 
     this.logger.warn(
-      `Failed to decrypt event from user '${remoteUserId}' (${remoteClientId}).\nError Code: '${errorCode}'\nError Message: ${
+      `Failed to decrypt event '${eventType}' from user '${remoteUserId}' with client '${remoteClientId}'.\nError Code: '${errorCode}'\nError Message: ${
```
Comment (gregor) [inline app/script/cryptography/CryptographyRepository.js]:
```
`Failed to decrypt event from client '${remoteClientId}' of user '${remoteUserId}' (${time}).\nError Code: '${errorCode}'\nError Message: ${
``

Comment (bennycode) [reply app/script/cryptography/CryptographyRepository.js]: updated

**Pull request merged**